build.sh

```diff
@@ -439,9 +439,9 @@ case "$COMMAND" in
         ;;
 
     "reset-ros-client-state")
-        rm -rf "~/Library/Application Support/xctest"
-        rm -rf "~/Library/Application Support/io.realm.TestHost"
-        rm -rf "~/Library/Application Support/xctest-child"
+        rm -rf ~/Library/Application Support/xctest
```

Comment:
This deletes the files `~/Library/Application` and `Support/xctest`, and -f makes it not complain about that neither of these exist. The space needs to be escaped.

Reply:
Thank you for catching this!